src/v2/Apps/Artist/Routes/Overview/Components/ZeroState.tsx

```diff
@@ -1,36 +1,31 @@
-import { ContextModule, Intent } from "@artsy/cohesion"
-import { Link, Message, Sans } from "@artsy/palette"
-import { useSystemContext } from "v2/Artsy"
+import { ContextModule } from "@artsy/cohesion"
+import { Clickable, Message, Sans } from "@artsy/palette"
 import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "v2/Components/FollowButton/FollowArtistButton"
 import React from "react"
-import { openAuthToFollowSave } from "v2/Utils/openAuthModal"
 
 export const ZeroState = props => {
-  const { mediator, user } = useSystemContext()
   const { is_followed, artist } = props
 
-  function handleOpenAuth() {
-    openAuthToFollowSave(mediator, {
-      entity: artist,
-      contextModule: ContextModule.worksForSaleRail,
-      intent: Intent.followArtist,
-    })
-  }
-
   return (
     <Message justifyContent="center" textSize="4">
       There aren’t any works available by the artist at this time.{" "}
       {!is_followed && (
         <>
           <FollowArtistButton
             artist={artist}
-            useDeprecatedButtonStyle={false}
-            user={user}
-            onOpenAuthModal={() => handleOpenAuth()}
+            contextModule={ContextModule.worksForSaleRail}
             render={({ name }) => {
               return (
                 <Sans size="4" color="black60" weight="regular">
-                  Follow <Link>{name}</Link>
+                  Follow{" "}
```

Comment:
do we need the {" "}?

Reply:
Yes, this allows us to have a space in between plain text and a component.